modules/user_manual/pages/files/webgui/navigating.adoc

```diff
@@ -69,41 +55,27 @@ sorting and managing your files.
 
 == Move Files
 
-You can move files and folders by dragging and dropping them into any
-directory.
+You can move files and folders by dragging and dropping them into any directory.
 
 === Play Videos
 
-You can play videos in ownCloud with the Media Viewer app, by clicking once on the file. 
-Please note, video streaming by the ownCloud Media Viewer depends on your web browser and the video’s format.
+You can play videos in ownCloud with the Media Viewer app, by clicking once on the file. Please note, video streaming by the ownCloud Media Viewer depends on your web browser and the video’s format.
 
 image:video_player_2.png[Watching a movie.]
 
 [NOTE]
 ====
-If your ownCloud administrator has enabled video streaming, and it doesn't work in your Web browser, it may be a browser-related issue. 
-See {moz-browser-compatibility-guide-url}[Mozilla’s Browser Compatibility Guide] for supported multimedia formats in Web browsers.
+If your ownCloud administrator has enabled video streaming, and it doesn't work in your Web browser, it may be a browser-related issue. See {moz-browser-compatibility-guide-url}[Mozilla’s Browser Compatibility Guide] for supported multimedia formats in Web browsers.
 ====
 
 === Settings
 
-The *Settings* gear icon, in the lower left-hand corner of the ownCloud
-window, allows you to show or hide hidden files in your ownCloud Web
-interface. These are also called dotfiles, because they are prefixed
-with a dot, e.g. `.mailfile`.
+The *Settings* gear icon, in the lower left-hand corner of the ownCloud window, allows you to show or hide hidden files in your ownCloud Web interface. These are also called dotfiles, because they are prefixed with a dot, e.g. `.mailfile`.
```

Comment:
```suggestion
The *Settings* gear icon in the lower left-hand corner of the ownCloud window allows you to show or hide hidden files in your ownCloud Web interface. These are also called dotfiles, because they are prefixed with a dot, e.g. `.mailfile`.
```